### PR TITLE
Create rocky-linux-8-3-rc1-release.md

### PR DESCRIPTION
**Pull request by lekoOwO**

### CODE REVIEW

File: blogs/rocky-linux-8-3-rc1-release.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+- Fastly
+- GitLab
+- Mattermost
+- MontaVista
+- NAVER Cloud
+- OpenDrives
+
+這僅僅只是一個開頭。
+
+我們的承諾是，對開源企業級作業系統的長期開發和支持──由社群而生，為了社群而做。
```
Comment (steveyiyo) [inline blogs/rocky-linux-8-3-rc1-release.md:81]: 我們的承諾對開源企業級作業系統的長期開發和支援 - 由社群而生，為了社群而做。

Comment (lekoOwO) [reply blogs/rocky-linux-8-3-rc1-release.md]: fixed

File: blogs/rocky-linux-8-3-rc1-release.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Rocky Linux 8.3 RC1 版本釋出
+date: '2021-04-30'
+description: 'Rocky 企業軟體基金會 (RESF) 很榮幸地在此宣布達成了 Rocky Linux 8.3 的一號候釋版的一般可用性。'
+posttype: '新聞'
+---
+
+Rocky 企業軟體基金會 (RESF) 很榮幸地在此宣布達成了 Rocky Linux 8.3 在 x86_64 和  aarch64 平台上的一號候釋版的一般可用性。
```
[the remark this answers is not in the document]
Comment (lekoOwO) [reply blogs/rocky-linux-8-3-rc1-release.md]: fixed

**Review by steveyiyo — APPROVED** [review]

**Comment by steveyiyo** [comment]
@pwtsai 看起來沒什麼問題了。你要不要再 review 一次

File: blogs/rocky-linux-8-3-rc1-release.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+當一個產品的測試版本有成為穩定版本的可能性時，我們稱這個版本為候釋版。
+
+候釋版本的釋出目的是為了讓社群測試與驗證 Rocky Linux 的功能並回報錯誤 (如果存在的話)。
+
+> 我要在哪裡下載候釋版?
+
+[https://rockylinux.org/download](https://rockylinux.org/download)
+
+> 要怎麼幫候釋版和未來的釋出版本鏡像?
```
Comment (pwtsai) [inline blogs/rocky-linux-8-3-rc1-release.md:24]: I think use "提供鏡像服務" would be better... what do you think?

File: blogs/rocky-linux-8-3-rc1-release.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+請寄 Email 給 [mirrors@rockylinux.org](mailto:mirrors@rockylinux.org) 並說明您的來意。
+
+> 我可以在生產環境中用候釋版本嗎?
+
+無論在**任何**情況下，您都不應該在生產環境中使用候釋版本。
```
[the remark this answers is not in the document]
Comment (pwtsai) [reply blogs/rocky-linux-8-3-rc1-release.md:30]: Sure~

File: blogs/rocky-linux-8-3-rc1-release.md (hunk 3 above)
Comment (lekoOwO) [reply blogs/rocky-linux-8-3-rc1-release.md]: fixed

**Comment by lekoOwO** [comment]
> Please kindly find my review response, thank you.

As the comment has all been resolved, would you kindly review this PR again? Thank you.

**Pull request merged**